get_dependencies.py

```diff
@@ -23,19 +23,39 @@ def download(url, filename):
                 sys.stdout.flush()
     sys.stdout.write('\n')
 
-
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if sys.platform == "linux":
+    print('[*] Downloading support files for linux ~ 108 MB')
```

Comment:
The Linux, Windows and Mac code is very similar, we shouldn't replicate it but reuse the code.